fiftyone/core/patches.py

```diff
@@ -720,6 +734,8 @@ def make_evaluation_patches_dataset(
             -   ``True`` to include all other fields
             -   ``None``/``False`` to include no other fields
         name (None): a name for the dataset
+        persistent (False): whether the dataset should persist in the database
```

Comment:
if patches (or any generated view) are now persistent, this might trigger unexpected dataset permissioning issues in teams/might require a hub change 

cc @j053y

Reply:
The point of the PR being cherry-picked here (which is already in `develop`) is to allow these utilities to be called directly, in which case they return ordinary datasets.

They are *not* the fancy generated datasets that occur when the undocumented `_generated=True` flag is pased.

Reply:
I guess the question is, is this a _patch_ or a new functionality?

Seems like something new to me.  I would suggest this should be included in, and tested in, 1.8.0, in my super uninformed analysis of new Other Fields.

Reply:
We already announced two of these things and recognized a community contribution in Slack, so this is a patch to correct the omission.

Reply:
Sure, new functionality instead of just a minor release seems fine.

Why do we object to putting this in a minor release and releasing it this week, and apologizing for our omission since this PR is happening in a public repo where anybody who cares already knows it's happened?

Semver means things.